### DISCUSSION TITLE
Preview hyperlink not rendering the same as final page

### DISCUSSION THREAD

**Discussion by vincent-polite** [requarks/wiki#3731] [Error / Bug Report]
**Describe the bug**
Markdown is supposed to fallback to actual HTML.  I attempted to create a hyperlink in this form to open a new tab/window:
`<a href="javascript:window.open(url, '_blank');">Popup</a>`

While in the editor, this works fine.  I can write the markup, move to the Preview pane, click the newly formed link, and the status preview and actual functionality work perfectly.

However, once I save the page, the link gets rewritten so it's just a basic `<a href="url">Popup</a>`

I tested caching by modifying other elements of the page, and the page is definitely getting updated, but the preview and live results are not similar.

**To Reproduce**
1. Edit any page
2. Create an <a href="javascript:window.open('', '_blank')">Link</a>
3.  Verify the link functionality in the preview window via hover and click
4.  Save the page
5.  Navigate to the page
6.  Attempt to hover over and click the link.
7.  For some reason it has been parsed/transformed into a normal hyperlink

**Expected behavior**
I expect the behavior of the preview pane to mirror what happens when you save the actual page.

**Host Info (please complete the following information):**
 - OS: Windows 2019
 - Wiki.js version: 2.1.113
 - Database engine: mssql 2019

**Comment by vincent-polite** [comment]
Hi, I updated my issue report because the relevant code wasn't wrapped in code block delimiters.  But I believe the parser is seeing the hyperlink portion of the window.open and rerendering it; changing the intention of the code.

What is odd is that in the live preview window, the javascript function works as intended, but once the page is published - doing a view source, the link has been rewritten.  I'm not sure how the parsing works, but something is being run through a different branch of code it seems.

**Comment by sbonaime** [comment]
Can you try this code which works on my wiki.js
`[test](https://wiki.js.org/){target="_blank"}  `
